src/components/SideBar/SideBar.module.scss

```diff
@@ -0,0 +1,38 @@
+@import 'variables.scss';
+
+.c-side-bar {
+  width: 15%;
+  height: 100%;
+
+  background: $color-green;
+
+  &__options {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__options-item {
+    padding: $space-s $space;
+    margin: $space-s;
+
+    text-decoration: none;
+    color: $color-black;
+    border-radius: $space-s;
+
+    &:hover {
+      background: $color-lightgray;
+      cursor: pointer;
+    }
+  }
+
+  &__bottom-line {
+    height: 1px;
+    background-color: $color-black;
+    margin: $space-s;
+  }
+
+  &__options-item-active {
+    background: $color-lightgray;
+    cursor: pointer;
+  }
```

Comment:
Since here i see active my guess this should be modifier of `__options-item` so `--active`